### ISSUE TITLE
Need to decide style, target audience for OSW text

### ISSUE THREAD

**Issue by jberkus** [theopensourceway/guidebook#142]
I was re-reading the [merged version of the Governance chapter](https://github.com/theopensourceway/guidebook/pull/100), and realizing that there's really no coherent style or voice for the text, which says to me that we don't have a core target audience.  For example, this paragraph:

> All open source projects operate in and with governance structures. The term "governance" carries multiple meanings in an organizational context. It can refer to regulatory matters or risk management issues, for example. More generally, though, it can also refer to a system of rules, roles, and procedures that determine how power in a project gets distributed.

In tone and style, that's aimed at middle-to-upper management for a large company or government agency.

The very next paragraph:

> Because open source projects are organizations, all projects feature governance structures. Some of these structures are more explicit than others. Some are more formal than others. But every project has them.

... is written for more of a general/developer audience.  

It might be a good idea to attack this issue sooner rather than later; the longer you put off deciding on a style and target audience, the more re-editing you'll need to do.

**Comment by bproffitt** [comment]
@jberkus Your points are well-made and will be taken under consideration. Please note that in our editorial process, an initially submitted chapter is merged _first_, and then undergoes multiple editorial passes, one of which is specifically aimed at modulating the "voice" of a given chapter to match that of the book at large. This particular chapter has not started that process yet, which would explain its rougher status.

**Comment by semioticrobotic** [comment]
This issue does open onto an important, broader conversation about the target audience/reader for the guidebook. To my knowledge, we've not had that discussion (or at least recorded the outcomes of that discussion anywhere). But it seems like one we'd want to have in advance of heavier editing for consistency of style and tone among chapters.

**Comment by bproffitt** [comment]
+1 for sure, it is an issue to discuss. I just didn't want to leave the impression that this chapter was anywhere near ready for prime time.

**Comment by jberkus** [comment]
Yes, and if you gave your writers a directive on tone/style/audience, you'd have less editing to do in the first place.
